oslabs-beta/ReactionTime
# homePage component merge to staging branch

## Description:

- Making a pull request to merge the homePage component from the home feature branch to the staging branch of OS-labs repo

- Merging all changes made to the homePage component

## Changes I Made:

- added homePage component to app

- cleaned up file structure to remove bloatware and redux dependencies

- removed unnecessary dependencies from package.json including husky

- added styled-components and material-ui as dependencies 

- including images needed for background and reactionTIme logo in resources folder

- added styled components 

- added extra subtitle and text components

## How to Test:

- npm start to run the app on local host 3000 to check that app is loading properly
